src/WebAPI/DependencyInjection.cs

```diff
@@ -39,6 +37,7 @@ string rulesGptCorsPolicy
         // TODO: Set CORS in Bicep
         var productionCorsUrls = new string[]
         {
+            "https://ashy-meadow-0a2bad900.3.azurestaticapps.net",
```

Comment:
I would move these to `appsettings.[env.]json`. Don't need to do that here but probably a good idea to add this to the backlog.

Reply:
I see there's already a TODO for this. Could do it in bicep too.

Reply:
I've raised an issue for this: https://github.com/SSWConsulting/SSW.Rules.GPT/issues/120